coordinator/src/services/validation/bundle-validator.ts

```diff
@@ -129,12 +129,12 @@ export function verifyCommonBundle(
     if (practitionerRole.practitioner && isReference(practitionerRole.practitioner)) {
       const practitioner = resolveReference(bundle, practitionerRole.practitioner)
       if (practitioner) {
-        const bodySDSUserID = getIdentifierValueForSystem(
+        const bodySDSUserID = getIdentifierValueOrNullForSystem(
           practitioner.identifier,
           "https://fhir.nhs.uk/Id/sds-user-id",
           'Bundle.entry("Practitioner").identifier'
         )
-        if (bodySDSUserID !== accessTokenSDSUserID) {
+        if (bodySDSUserID && bodySDSUserID !== accessTokenSDSUserID) {
```

Comment:
Could you please add a variable or a comment about what we're checking here? E.g. `const isConditionWeAreChecking = bodySDSUserID && bodySDSUserID !== accessTokenSDSUserID`